src/main/java/seedu/address/logic/commands/util/CommandList.java

```diff
@@ -36,12 +38,39 @@
     public static final String UNFAVOURITE_COMMAND = UnfavouriteCommand.COMMAND_WORD;
 
     /**
-     * Returns the list of Commands available to users
+     * Returns the list of {@code COMMAND_WORD}s as Strings
+     *
+     * The default COMMAND_WORD is in lowercase format. We do some cleaning here to capitalize the COMMAND_WORD.
+     */
+    public static final ObservableList<String> COMMANDS = FXCollections.observableArrayList(
+            Arrays.asList(ADD_COMMAND, ALIAS_COMMAND,
```

Comment:
Perhaps instantiating a static list of all the command words at line 39 would be more maintainable since COMMANDS and COMMANDS_WORDS_LIST both uses the list of command words?